### PR TITLE
Cross Mio Updates for Pioneer 17

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Added the Pioneer 17 cruise to CruiseInformation.csv (for now the best place to confirm that is the draft cruise plan in Vault: 3204-01701

Added the following to sensor_bulk_load.csv:

- 5 new FDCHP R3 anemometers (2 received in 2020 but hadn't been added yet, 3 received recently)
Also added the received/aquisition date for the R3 anemometer that was already in bulk load (SN 203902)
*FDCHPs can be confirmed via WHOI_Asset_Tracking.xlsx in Vault

- 2 new hydrogen sensors (OSSM-14 config)
- 1 new MOPAK sensor (CNSM-15 config)

### CODE REVIEW

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -1291,7 +1291,12 @@ CGINS-FDCHPA-70601,,Sensor,0,Direct Covariance Flux: FDCHP Series A,ANEMOMETER W
 CGINS-FDCHPA-71403,,Sensor,0,Direct Covariance Flux: FDCHP Series A,ANEMOMETER WINDMASTER PRO,Gill,WINDMASTER PRO,171403,,20170505,,,Woods Hole Oceanographic Institution
 CGINS-FDCHPA-74102,,Sensor,0,Direct Covariance Flux: FDCHP Series A,ANEMOMETER WINDMASTER PRO,Gill,WINDMASTER PRO,174102,,20180109,,,Woods Hole Oceanographic Institution
 CGINS-FDCHPA-75008,,Sensor,0,Direct Covariance Flux: FDCHP Series A,ANEMOMETER WINDMASTER PRO,Gill,WINDMASTER PRO,175008,,,,,Woods Hole Oceanographic Institution
-CGINS-FDCHPB-03902,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,203902,,,,,Woods Hole Oceanographic Institution
+CGINS-FDCHPB-03902,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,203902,,20200924,,,Woods Hole Oceanographic Institution
+CGINS-FDCHPB-03903,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,203903,,20201020,,,Woods Hole Oceanographic Institution
+CGINS-FDCHPB-03904,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,203904,,20201020,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:1296]: Acquisition date for these should be 11/03/2020, per WHOI_Asset_Tracking and Breago.

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -1291,7 +1291,12 @@ CGINS-FDCHPA-70601,,Sensor,0,Direct Covariance Flux: FDCHP Series A,ANEMOMETER W
...
+CGINS-FDCHPB-11802,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,211802,,20210824,,,Woods Hole Oceanographic Institution
+CGINS-FDCHPB-12802,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,212802,,20210824,,,Woods Hole Oceanographic Institution
+CGINS-FDCHPB-12803,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,212803,,20210824,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:1299]: Acquisition date for these should be 09/01/2021 per Breago, WHOI_Asset_Tracking has 09/02/2021.

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 3]
```diff
@@ -1570,6 +1575,8 @@ CGINS-HYDGN0-49891,,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instrume
 CGINS-HYDGN0-51407,,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,M2A51407,,,,,
 CGINS-HYDGN0-99999,,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,UNKNOWN,,,,Placeholder for unknown S/N,
 CGINS-HYDGN0-Z5971,,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,M2A6Z5971,,,,,Woods Hole Oceanographic Institution
+CGINS-HYDGN0-Z5974,,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,M2A6Z5974,,,,,Woods Hole Oceanographic Institution
+CGINS-HYDGN0-Z5975,,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,M2A6Z5975,,,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:1579]: Looks good.

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 4]
```diff
@@ -1840,6 +1847,7 @@ CGINS-MOPAK0-11364,OL000339,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Mo
 CGINS-MOPAK0-11366,,Sensor,0,3-Axis Motion Pack: MOPAK,,MicroStrain,3DM-GX3-25,6223-4220-11366,,,,,Woods Hole Oceanographic Institution
 CGINS-MOPAK0-11838,,Sensor,0,3-Axis Motion Pack: MOPAK,,MicroStrain,3DM-GX3-25,6223-4220-11838,,,,,
 CGINS-MOPAK0-11839,,Sensor,0,3-Axis Motion Pack: MOPAK,,MicroStrain,3DM-GX3-25,6223-4220-11839,,,,,
+CGINS-MOPAK0-11841,,Sensor,0,3-Axis Motion Pack: MOPAK,,MicroStrain,3DM-GX3-25,6223-4220-11841,,,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:1850]: Looks good.

File: cruise/CruiseInformation.csv [hunk 5]
```diff
@@ -263,3 +263,4 @@ TN393,Thomas G. Thompson,2021-08-01T00:00:00,2021-09-02T23:59:59,Cabled (rvdata)
 TS20210809,Tenacious II,2021-08-09T01:00:00,2021-08-09T23:59:59,Pioneer gliders cp_559 and cp_564 recoveries
 TS20210903,Tenacious II,2021-09-03T14:00:00,2021-09-04T10:00:00,Pioneer 16d glider deployments
 TN394,Thomas G. Thompson,2021-09-08T21:00:00,2021-09-22T21:50:00,Endurance 15 moorings profilers and gliders
+AR61,Neil Armstrong,2021-10-26T00:00:00,2021-11-15T23:59:00,Pioneer 17
```
Comment (snwhite) [inline cruise/CruiseInformation.csv:266]: Looks good.

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 1 above)
Comment (rtravis-whoi) [reply bulk/sensor_bulk_load-AssetRecord.csv:1296]: corrected

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 2 above)
Comment (rtravis-whoi) [reply bulk/sensor_bulk_load-AssetRecord.csv:1299]: corrected

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 6]
```diff
@@ -1291,7 +1291,12 @@ CGINS-FDCHPA-70601,,Sensor,0,Direct Covariance Flux: FDCHP Series A,ANEMOMETER W
...
-CGINS-FDCHPB-03902,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,203902,,,,,Woods Hole Oceanographic Institution
+CGINS-FDCHPB-03902,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,203902,,20201103,,,Woods Hole Oceanographic Institution
+CGINS-FDCHPB-03903,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,203903,,20201103,,,Woods Hole Oceanographic Institution
+CGINS-FDCHPB-03904,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,203904,,20201103,,,Woods Hole Oceanographic Institution
+CGINS-FDCHPB-11802,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,211802,,20210901,,,Woods Hole Oceanographic Institution
+CGINS-FDCHPB-12802,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,212802,,20210901,,,Woods Hole Oceanographic Institution
+CGINS-FDCHPB-12803,,Sensor,0,Direct Covariance Flux: FDCHP Series B,ANEMOMETER R3-50,Gill,R3-50,212803,,20210901,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:1299]: Looks good.

**Pull request merged**